ui/src/criteria/attribute.tsx

```diff
@@ -81,52 +85,160 @@ function AttributeEdit(props: AttributeEditProps) {
   const hintDisplayName = (hint: tanagra.EnumHintValue) =>
     hint.displayName || "Unknown Value";
 
-  const enumHintValues = underlay.entities
+  const attributeFilterHint = underlay.entities
     .find((g) => g.name === underlay.primaryEntity)
-    ?.attributes?.find((attribute) => attribute.name === props.data.attribute)
-    ?.attributeFilterHint?.enumHint?.enumHintValues;
+    ?.attributes?.find(
+      (attribute) => attribute.name === props.data.attribute
+    )?.attributeFilterHint;
+
+  const year = new Date().getFullYear();
+
+  if (
+    attributeFilterHint?.integerBoundsHint !== null &&
+    attributeFilterHint?.integerBoundsHint !== "undefined"
+  ) {
+    const integerBoundsHint = attributeFilterHint?.integerBoundsHint;
+
+    if (
+      integerBoundsHint?.max !== null &&
+      typeof integerBoundsHint?.max !== "undefined" &&
+      integerBoundsHint?.min !== null &&
+      typeof integerBoundsHint?.min !== "undefined"
+    ) {
```

Comment:
@tjennison-google Tim. Could you please help me think about if there is another smarter or efficient way to rewrite the 2 if sentences?

Reply:
There's an `isValid` function in `util/valid.tsx` that checks for undefined and null if that's what you're looking for.

Reply:
Oh! Good catch!